tests/storage/export.go

```diff
@@ -187,7 +187,7 @@ var _ = SIGDescribe("Export", func() {
 	createSourcePodChecker := func(pvc *k8sv1.PersistentVolumeClaim) *k8sv1.Pod {
 		volumeName := pvc.GetName()
 		podName := "download-pod"
-		pod := tests.RenderPod(podName, []string{"/bin/sh", "-c", "sleep 360"}, []string{})
+		pod := tests.RenderPrivilegedPod(podName, []string{"/bin/sh", "-c", "sleep 360"}, []string{})
```

Comment:
Do they fail as not privileged? I recall getting by without that with green lanes

Reply:
I just forgot to change it, let me re-push, and let's see.

Reply:
Yep, it does fail.

Reply:
thats weird, here is a passing lane where I didn't use privileged
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/8645/pull-kubevirt-e2e-k8s-1.24-sig-storage/1584153649972514816